replay/models/nn/sequential/sasrec_with_llm/utils.py

```diff
@@ -0,0 +1,144 @@
+# src/utils.py
+
+import random
+import numpy as np
+import torch
+import json
+
+from torch import nn
+import torch.nn.functional as F
+
+
+def set_seed(seed):
```

Comment:
Are functions `set_seed`, `init_criterion_reconstruct`, `calculate_recsys/guide_loss` used in the code?